pages/_app.tsx

```diff
@@ -0,0 +1,36 @@
+import {
```

Comment:
I read up a bit and tried myself. This is also always called in case of SSR, so we can probably remove the withOptimizely HOC. That is not needed anymore. we can initialize optimizely here just like we do for a normal react application. The only complexity is to detect and provide datafile / sdk key depending on if its server or client